recipes/eccodes-python/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "eccodes-python" %}
+{% set version = "0.9.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 587dd828ee5e13b6a7d35a362d6fd903d914dd0422f4010af95e8f45738e9143
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - numpy
+    - python
+
+test:
+  imports:
+    - eccodes
```

Comment:
```suggestion
    - gribapi
    - eccodes
```

Reply:
eccodes is a superset of grib_api and is therefore not needed

Reply:
But this package provides a library which can be accessed using `import gribapi`.

Reply:
This is done for legacy applications. Is is only an alias for 'import eccodes'.

Reply:
I see now what you mean. Yes, we can also test that this works.